Wabbajack.App.Blazor/Components/StepLogger.razor

```diff
@@ -0,0 +1,20 @@
+﻿@using Wabbajack.App.Blazor.Pages
+
+@namespace Wabbajack.App.Blazor.Components
+
+<div id="step-logger">
+    @foreach (var step in value.StatusStep.Take(ShownSteps))
+    {
+        <div class="step">@step</div>
+    }
+</div>
+
+@code {
+    [Parameter]
+    public int ShownSteps { get; set; }
+    [Parameter]
+    public bool Reverse { get; set; }
+
+    [CascadingParameter]
+    public Installing value { get; set; }
+}
```

Comment:
I saw the `<CascadingValue Value=this>` below, I would just remove this component and inline this inside the `Installing` page. There is no reason to have this as a component, it just degrates performance. This component can't even be re-used because you used `[CascadingParamtere] public Installing value { get; set; }` so just inlined this entire thing.